src/Form/Helpers/Themes.php

```diff
@@ -0,0 +1,21 @@
+<?php
+namespace Give\Form\Themes;
+
+use Give\Form\Theme;
+use Give\Form\Themes;
+
+/**
+ * Register core themes
+ *
+ * @since 2.7.0
+ */
+function registerDefaults() {
+	/**
+	 * Register themes
+	 */
+	$themes = require GIVE_PLUGIN_DIR . 'src/Form/Config/Themes/Load.php';
```

Comment:
So is there a context wherein you want to have _every_ theme configuration loaded into memory? Or is it more likely that you just want to load the current active one?

Reply:
@JasonTheAdams Good catch. I updated logic to prevent unnecessary memory use: https://github.com/impress-org/givewp/compare/2d0997eecf7e...33580c67784f

Reply:
Let me rethink above changes

Reply:
this will be resolve in https://github.com/impress-org/givewp/pull/4518